pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1559,10 +1554,13 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 
 			// if UseEmulation unset and at least one NIC model is virtio,
 			// /dev/vhost-net must be present as we should have asked for it.
+			mq := vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue
```

Comment:
Can you please use more meaningful name than mq?

Reply:
It is used to avoid long lines (or breaking lines).
It has no usage beyond these 2 lines, so I find this redundant.

Do I need to use full names here because it is not clear/readable what it represents?

Reply:
Up to you, I won't insist. I believe full name will be more readable.

Reply:
The full name has no meaning here, is it just a temp variable.
I will embed this into an `if` to make sure no one will try to re-use it beyond its scope here. Hope that will resolve this saga.